content/includes/header.html

```diff
@@ -34,6 +34,9 @@
 <meta content="{{ tag }}" property="article:tag">
 {% endfor %}
 {% endif %}
+{% if title == 'Education Benefits Application Process' or title == 'Education Benefits Eligibility' %}
+<link rel="import" href="/education/wizard/index.html">
```

Comment:
Thanks @jbalboni, using liquid tags doesn't work as expected. There are two types of includes syntax used in our content (with/without quotes):
using with quotes, yields:
<img width="553" alt="screen shot 2017-08-22 at 9 37 40 am" src="https://user-images.githubusercontent.com/16051172/29576751-dba5e114-871d-11e7-9081-1471b6957af5.png">

using  without, yields:
<img width="538" alt="screen shot 2017-08-22 at 9 37 53 am" src="https://user-images.githubusercontent.com/16051172/29576753-de50b330-871d-11e7-9857-f150c0792e01.png">

The linked documentation was the only I could find to provide insight on why using liquid was not rendering in the collection pages.